### PR TITLE
add study short citation to epi/animal/invitro bulk cleaning interfaces

### PR DESCRIPTION
**Pull request by shanethacker**
Update to the data cleanup views, adding short citation names to individual value listings. Addresses #134

### CODE REVIEW

**Comment by shanethacker** [comment]
Messed that one up. Let me create another one.

File: .gitignore [hunk 1]
```diff
@@ -1,45 +0,0 @@
-
```
Comment (shapiromatron) [inline .gitignore]: can this file be restored?

File: project/epi/serializers.py [hunk 2]
```diff
@@ -212,10 +212,13 @@ class Meta:
...
+    study_short_citation = serializers.SerializerMethodField()
 
     class Meta:
         model = models.Outcome
-        cleanup_fields = model.TEXT_CLEANUP_FIELDS
+        cleanup_fields = ('study_short_citation',) + model.TEXT_CLEANUP_FIELDS
         fields = cleanup_fields + ('id', )
 
+    def get_study_short_citation(self, obj):
+        return obj.study_population.study.short_citation
```
Comment (shapiromatron) [inline project/epi/serializers.py]: add blank line to separate class from  add_serializer method

Comment (shapiromatron) [reply project/epi/serializers.py]: @shanethacker a few tweaks, thanks for submitting!

**Pull request closed without merging**